wmn-data.json

```diff
@@ -5177,6 +5177,16 @@
         "known" : ["spicerex", "rod-it"],
         "cat" : "tech"
        },
+      {
+        "name" : "SPOJ",
+        "uri_check" : "https://www.spoj.com/users/{account}/",
+        "e_code" : 200,
+        "e_string" : "<li>Profile</li>",

```

Comment:
![image](https://github.com/user-attachments/assets/7f6d8c92-9cf6-41b2-999f-63ab2a7d0d29)

I'm not seeing the `e_string` in the response from the server for good/valid accounts. See the image where I visited the defrager user (your known user) and it responded with a valid user profile page. I then looked at the response and searched for the `e_string` and it was not found.

Reply:
Hi, I must have searched for e_string while I was logged in, that would have produced the `Profile` element.
Corrected it.